projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -416,6 +416,7 @@ if (environment.cpq) {
         enableClaimCustomerCouponWithCodeInRequestBody: false,
         enableReadDomainValuesOnDemand: true,
         opfEnablePreventingFromCheckoutWithoutEmail: true,
+        enableProductRouteShorterMatcher: true,
```

Comment:
Becasue this flag is true in storefrontapp, we can remove the config from the app.module:
https://github.com/SAP/spartacus/blob/79f864b2f797658cb3e6e518b93f48dd9cedcf20/projects/storefrontapp/src/app/app.module.ts#L78